src/app/client/projects/desktop/src/app/modules/content-consumption/components/content-player/content-player.component.ts

```diff
@@ -159,10 +160,15 @@ export class ContentPlayerComponent implements AfterViewInit, OnChanges, OnInit,
    * Emits event when content starts playing and end event when content was played/read completely
    */
   loadPlayer() {
-    if (_.includes(this.router.url, 'browse')) {
+    const downloadStatus = _.has(this.playerConfig, 'metadata.desktopAppMetadata') ?
+    !_.has(this.playerConfig, 'metadata.desktopAppMetadata.isAvailable') ||
```

Comment:
Define all these string in the CONSTANT variable & use it. This is to avoid hardcoded strings in the code.